.github/workflows/build-pkg.yml

```diff
@@ -24,16 +27,51 @@ jobs:
         id: print-version
         run: |
           echo "version=$(git -C brew describe --tags --always)" > $GITHUB_OUTPUT
+      - name: Install the Apple certificate
+        env:
+          PKG_BUILD_CERTIFICATE_BASE64: ${{ secrets.PKG_BUILD_CERTIFICATE_BASE64 }}
+          PKG_BUILD_P12_PASSWORD: ${{ secrets.PKG_BUILD_P12_PASSWORD }}
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+        run: |
+          # create variables
+          CERTIFICATE_PATH=$RUNNER_TEMP/build_certificate.p12
+          KEYCHAIN_PATH=$RUNNER_TEMP/app-signing.keychain-db
+          
+          # import certificate
+          echo -n "$PKG_BUILD_CERTIFICATE_BASE64" | base64 --decode -o $CERTIFICATE_PATH
+
+          # create temporary keychain
+          security create-keychain -p "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+          security set-keychain-settings -lut 21600 $KEYCHAIN_PATH
+          security unlock-keychain -p "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+          
+          # import certificate to keychain
+          security import $CERTIFICATE_PATH -P "$PKG_BUILD_P12_PASSWORD" -A -t cert -f pkcs12 -k $KEYCHAIN_PATH
+          security list-keychain -d user -s $KEYCHAIN_PATH
       - name: Build package
         run: |
          pkgbuild --root brew \
                   --scripts brew/package/scripts \
                   --install-location "$TMP_PATH" \
                   --identifier "$IDENTIFIER" \
+                  --sign "$SIGNING_IDENTITY" \
                   --min-os-version "$MIN_OS" \
                   --filter .DS_Store \
+                  --filter "(.*)/Library/Homebrew/test/support/fixtures/" \
```

Comment:
@MikeMcQuaid any suggestions how to deal with these better?

Reply:
@SMillerDev This seems fine with me but the package itself will need to do a `git checkout master` and/or `git reset --hard origin/master` after installation on disk or `brew doctor` will complain immediately.

Relatedly, that'd be good CI to add here at some point (doesn't need to block this PR): actually install the package and run `brew doctor` on the result to ensure it's setup correctly.

Reply:
The problem with that is finding a CI for macOS without brew pre-installed.

Reply:
@SMillerDev it's pretty easy to remove it: https://github.com/MikeMcQuaid/strap/blob/942c15c2f5ee656c4d97f4b63944f6ef641a584a/.github/workflows/tests.yml#L38-L43